lib/Mojo/WebSocketProxy/Dispatcher.pm

```diff
@@ -123,6 +124,9 @@ sub on_message {
     my $req_storage = {};
     $req_storage->{args} = $args;
 
+    # any further key-val pairs can be added here in request storage context object
+    $req_storage->{logger_context} = {correlation_id => UUID::Tiny::create_UUID_as_string(UUID::Tiny::UUID_V4)};
```

Comment:
we probably can move the context creation to the logger itself. if we found that there is no performance hit by keeping this constructor here.